contracts/SimpleMarginTrading.sol

```diff
@@ -8,12 +8,12 @@ import "@0x/contracts-utils/contracts/src/LibSafeMath.sol";
 
 // interfaces
 import "@0x/contracts-erc20/contracts/src/interfaces/IERC20Token.sol";
-import "@0x/contracts-exchange/contracts/src/interfaces/IExchange.sol";
+import "@0x/contracts-exchange/contracts/src/interfaces/Iexchange.sol";
 import "@0x/contracts-asset-proxy/contracts/src/interfaces/IAssetData.sol";
 import "@0x/contracts-erc20/contracts/src/interfaces/IEtherToken.sol";
 import "./interfaces/ICERC20.sol";
-import "./interfaces/ICETH.sol";
-import "./interfaces/IComptroller.sol";
+import "./interfaces/Iceth.sol";
```

Comment:
Nit: This is the spelling that Compound uses. Let's use something more like that (`ICEth` or `ICEther`) https://github.com/compound-finance/compound-protocol/blob/master/contracts/CEther.sol